R/append_mc_name.R

```diff
@@ -0,0 +1,14 @@
+#' @title Append "mc" to Appropriate Column names
+#' @description Ensure names for MC columns in fit have the suffix "_mc"
+#' @param .data Dataframe/tibble whose columns include "MC" calculations.
+#' @return \code{.data}, with column names appended appropriately.
+#' @export 
```

Comment:
> Yeah I wouldn't mind exporting it, just in case I have to change the column names of an old model and don't want to run the whole model again. Thanks, forgot to re-generate docs, done now